src/main/java/picard/analysis/directed/CollectTargetedMetrics.java

```diff
@@ -98,6 +102,11 @@ protected abstract COLLECTOR makeCollector(final Set<MetricAccumulationLevel> ac
     @Option(doc="Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.", optional = true)
     public int SAMPLE_SIZE=10000;
 
+    @Option(doc="Output for Theoretical Sensitivity metrics.  Default is null.", optional = true)
+    public File THEORETICAL_SENSITIVITY_OUTPUT;
+
+    @Option(doc="Allele fraction to run theoretical sensitivity on.", optional = true)
+    public List<Double> ALLELE_FRACTION = new LinkedList<>(Arrays.asList(0.001, 0.005, 0.01, 0.02, 0.05, 0.1, 0.2, 0.3, 0.5));
```

Comment:
The .put() is used in the command line parsing, so I can't easily wrap it later.